ocaml/xapi/message_forwarding.ml

```diff
@@ -4285,6 +4285,7 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       let pool = Db.Pool.get_all ~__context |> List.hd in (* assumes 1 pool in DB *)
       Xapi_pool_helpers.with_pool_operation ~__context ~self:pool ~doc:"Cluster.create" ~op:`cluster_create
         (fun () ->
+           Helpers.assert_rolling_upgrade_not_in_progress ~__context;
```

Comment:
I think this should be part of xapi_pool_helpers.ml, in the valid_operations check for `cluster_create`. That way creating a cluster should get greyed out in XenCenter.